src/Microsoft.Framework.PackageManager/Properties/AssemblyInfo.cs

```diff
@@ -4,5 +4,6 @@
 using System.Reflection;
 using System.Runtime.CompilerServices;
 
+[assembly: InternalsVisibleTo("Microsoft.Framework.PackageManager.FunctionalTests")]
```

Comment:
I could change this to unit tests, since it is a weird mix between unit and functional, but we need internals visible otherwise the only way to test this is to compromise the users current installs